.github/issue_template.md

```diff
@@ -1,16 +1,24 @@
-For bugs, please fill out the three following sections completely. For the ticket to be resolved, you'll want to provide as much information as possible.
+Issue tracker is **ONLY** used for reporting bugs. NO NEW FEATURE REQUESTS ACCEPTED. For new features navigate to [Request new feature](https://github.com/reactioncommerce/reaction-feature-requests).
```

Comment:
For new features, add a issue in our [Reaction Feature Requests](https://github.com/reactioncommerce/reaction-feature-requests) repository.